src/endpoints/ui.rs

```diff
@@ -169,25 +210,16 @@ async fn parse_form<T: for<'de> Deserialize<'de>>(
     let mut qs = String::new();
     while let Some(field) = form.next_field().await? {
         let name = field.name().ok_or(ParseFormError::NameMissing)?;
-        let name = urlencoding::encode(name);
+        let name = urlencoding::encode(name).to_string();
```

Comment:
Why is `.to_string()` required here?

Reply:
Yes, kind of. It is due to `field` being moved there and the `name` variable being needed later on. (Previously, the ordering of the code was different.)
An alternative would probably, to not move `field` and construct `name` afterwards, as it was done previously.